### PR TITLE
Update template example for new syntax

### PR DESCRIPTION
**Pull request by mvermaes**
Change `filename` to `template` for template provider new syntax.

### CODE REVIEW

File: website/source/docs/providers/template/index.html.markdown [hunk 1]
```diff
@@ -25,7 +25,7 @@ Use the navigation to the left to read about the available resources.
 ```
 # Template for initial configuration bash script
 resource "template_file" "init" {
-	filename = "init.tpl"
+	template = "init.tpl"
```
Comment (jen20) [inline website/source/docs/providers/template/index.html.markdown]: This change is not quite correct: the deprecated version uses `filename = "init.tpl"`, but the new version takes the *content* of the template in the variable which also allows inline templates in the form of heredocs (for example). Consequently in this example you'd likely use a the file interpolation function like this: `template = "${file("init.tpl")}".

**Comment by jen20** [comment]
Thanks for submitting this @mvermaes! I've left some comments in line - could you let me know what you think? We may want to add a note explaining the warning which comes from 0.6.7 as well.

**Comment by mvermaes** [comment]
Oops, sorry about that. I should have read the docs before assuming it was just a change in variable name ... have added the correct format now.

I noticed there's a warning about the change already in the resource page at https://terraform.io/docs/providers/template/r/file.html. Let me know if you'd like me to add one to this page as well, or if the resource one is sufficient.

**Comment by jen20** [comment]
Hi @mvermaes - thanks for updating. I squashed your commits locally and merged, so the PR doesn't appear merged, but the work is on `master` and you still have credit for it. Thanks!

**Pull request closed without merging**